### PR TITLE
LPS-84683 Removing an attachment makes the changes of body disappear

### PR DESCRIPTION
**Pull request by IstvanD**
Hello @ambrinchaudhary 

Could you please review this pull request?
It introduced an ajax based attachment removal logic, that prevents the page being reloaded.

Thank you and best regards,
István

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/page_attachment_action.jsp [hunk 1]
```diff
@@ -65,18 +65,12 @@ WikiPage wikiPage = WikiPageAttachmentsUtil.getPage(attachmentsFileEntry.getFile
...
-					<portlet:param name="nodeId" value="<%= String.valueOf(wikiPage.getNodeId()) %>" />
-					<portlet:param name="title" value="<%= wikiPage.getTitle() %>" />
-					<portlet:param name="fileName" value="<%= HtmlUtil.unescape(attachmentsFileEntry.getTitle()) %>" />
-				</portlet:actionURL>
-
-				<liferay-ui:icon-delete
-					trash="<%= trashHelper.isTrashEnabled(scopeGroupId) %>"
-					url="<%= deleteURL %>"
-				/>
+				<div class="delete-attachment" data-cmd="<%= trashHelper.isTrashEnabled(scopeGroupId) ? Constants.MOVE_TO_TRASH : Constants.DELETE %>" data-fileName="<%= attachmentsFileEntry.getTitle() %>" data-nodeId="<%= String.valueOf(wikiPage.getNodeId()) %>" data-rowId="<%= attachmentsFileEntry.getFileEntryId() %>" data-title="<%= wikiPage.getTitle() %>">
```
Comment (ambrinchaudhary) [inline modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/page_attachment_action.jsp]: Hi @IstvanD, 
don't we need here the `data-ticketKey` param?

Comment (IstvanD) [reply modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/page_attachment_action.jsp]: Hi @ambrinchaudhary 

I was thinking about it, but I could not manage to create the ticket per Wikipage, only per attachment, as ```page_attachment_action.jsp``` is rendered on each attachment.

Comment (ambrinchaudhary) [reply modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/page_attachment_action.jsp]: ok, I see... but I am just wondering, is that param really necessary? I don't see it in the previous actions url's?  What do you think @robertoDiaz ?

**Comment by ambrinchaudhary** [comment]
Thanks @IstvanD for your work. I will check with @sergiogonzalez the `ticketKey` issue.

**Pull request closed without merging**